src/hs70.jl

```diff
@@ -0,0 +1,37 @@
+# HS problem 70 in NLS format
+#
+#   Source:
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag Berlin Heidelberg, 1981
+#   10.1007/978-3-642-48320-2
+#
+# A. Montoison, Montreal, 06/2018.
+
+export hs70
+
+"Hock-Schittkowski problem 70 in NLS format"
+function hs70()
```

Comment:
This is a name clash with a problem in `OptimizationProblems`. How should we deal with that?

Reply:
And I think we agreed that all problems would accept arguments. If a problem only comes in one size, the function should have prototype `function hs70(args...)`.

Reply:
We could change the name into ```hs*_NLS```? There is also ```NZF1``` that is present here and in```OptimizationProblems```.

And ```args...``` option is only present on the other repository. I think it's a good idea to add it here too for  ```hs``` and  ```tp``` problems.

Reply:
These NLS problems also work as NLP problems, but with additional overhead, so we could have only NLS problems. If we finish the [merge between NLP and NLS](https://github.com/JuliaSmoothOptimizers/NLPModels.jl/pull/109), maybe this would improve?